hamster_gtk/preferences/preferences_dialog.py

```diff
@@ -80,6 +80,11 @@ def __init__(self, parent, app, initial, *args, **kwargs):
                 ('autocomplete_split_activity',
                  (_("Autocomplete activities and categories separately"),
                   HamsterSwitch())),
+                ('tracking_show_recent_activities',
+                 (_("Allow tracking based on recent activities."),
```

Comment:
This does not sound clear to me. I would choose something like “Show recent activities for quickly starting tracking”